src/Evolution/Systems/Cce/BoundaryData.cpp

```diff
@@ -484,6 +485,10 @@ void null_vector_l_and_derivatives(
     const tnsr::I<DataVector, 3>& shift,
     const tnsr::I<DataVector, 3>& worldtube_normal) {
   const size_t size = get(lapse).size();
+  CAPTURE_FOR_ERROR(lapse);
+  CAPTURE_FOR_ERROR(dt_lapse);
+  CAPTURE_FOR_ERROR(shift);
+  CAPTURE_FOR_ERROR(dt_shift);
```

Comment:
[optional] Would it also be useful to capture the `denominator` below before dividing by it?

Reply:
I'll do that and ping you once it's done